newsroom/topics/views.py

```diff
@@ -37,6 +36,8 @@ async def get_topics(args: RouteArguments, params: None, request: Request) -> Re
 )
 async def post_topic(request: Request) -> Response:
     """Creates a user topic"""
+    # Import here to prevent circular imports
+    from newsroom.notifications import push_user_notification, push_company_notification
```

Comment:
This should be fixable in [newsroom/notifications/utils.py](https://github.com/superdesk/newsroom-core/blob/async/newsroom/notifications/utils.py#L13), by replacing `WireSearchServiceAsync` with the type, and using `find_by_ids`

Reply:
I tried but it is not working

Reply:
I see, then it is something else as well causing the circular imports, we can leave as is for now then.
Ideally it would be good to fix these circular imports, as importing modules inside a function is not good practice and I would prefer we avoid where possible